internal/githelpers.go

```
@@ -59,19 +59,20 @@ func ParseBranchname(branchString string) (remote, branchname string) {
 	return branchString, ""
 }
 
+// DeleteBranch deletes a remote branch by shelling out to git.
+// We use shell commands instead of go-git for deletion to avoid complex authentication
+// handling. The go-git library has significant limitations with various authentication
+// methods (SSH keys, tokens, etc.). See: https://github.com/go-git/go-git/issues/28
 func DeleteBranch(repo *git.Repository, remote, branchShortName string) error {
```

golangci-lint (2.5.0)

[warning] unused-parameter: parameter 'repo' seems to be unused, consider removing or renaming it as _

(revive)